pkg/image/client.go

```diff
@@ -97,6 +116,21 @@ func (i *Client) CreateOrUpdateImage(ctx context.Context, image *images.NodeImag
 		}
 		return err
 	}
+	return nil
+}
+
+func (i *Client) AddRelease(ctx context.Context, image string) error {
```

Comment:
Similar as I mentioned above, maybe we can pass the `NodeImage` instead, so we don't have to fetch it from the API yet again.

Reply:
yeah true